main.js

```diff
@@ -121,8 +121,7 @@ app = {
 				// check to see if it's new
 				if (hackProgress === newHackProgress) {
 					// the bar hasn't moved
-					log("* Progress bar hasn't moved, waiting");
-					// TODO: make this an automatic process
+					app.restart();	//restart if not work
```

Comment:
yes same for me it's for the begun next i will go debug this cause whitout this when the progress bar make a rollback due to target make +5 the bot will enter in infinity wait 
(sorry i'm french)
but try without you will seen what i want say